### PR TITLE
some minor fixes from testing

### PR DESCRIPTION
**Pull request by maleck13**
adds some fixes plus support in OSCP for tls serving

### CODE REVIEW

**Comment by maleck13** [comment]
@david-martin the change to use tls stopped the server from working inside the docker container. I think we might be able to use the annotation on the route to get a generated cert from openshift

File: cmd/mobile-server/main.go [hunk 1]
```diff
@@ -101,10 +102,13 @@ func main() {
 
 	handler := web.BuildHTTPHandler(router, mwAccess)
 	http.Handle("/", handler)
-	logger.Info("starting server on port " + *port)
-	if err := http.ListenAndServeTLS(*port, *cert, *key, nil); err != nil {
+	logger.Info("starting server on port "+*port, *key, *cert, "target namespace is ", *namespace)
+	if err := http.ListenAndServe(*port, nil); err != nil {
```
Comment (david-martin) [inline cmd/mobile-server/main.go]: @maleck13  maybe we should add an arg for using tls or not.
For local dev tls is needed for the oauth flow.
When running in openshift, we could do edge termination

Comment (maleck13) [reply cmd/mobile-server/main.go]: I have added changes to allow us to use a cert generated by oscp now see latest commit

**Comment by maleck13** [comment]
@philbrookes take a look also as it will likely effect you

**Pull request closed without merging**